src/main/kotlin/com/sourcegraph/cody/commands/ui/CommandsTabPanel.kt

```diff
@@ -100,9 +102,10 @@ class CommandsTabPanel(
     buttons[actionId] = button
 
     button.layout = BorderLayout()
-    button.border = BorderFactory.createEmptyBorder(0, 5, 0, 10)
     ActionManagerEx.getInstanceEx().getAction(actionId).shortcutSet.shortcuts.firstOrNull()?.let {
-      button.add(JLabel(KeymapUtil.getShortcutText(it)), BorderLayout.LINE_END)
+      val shortcutLabel = JLabel(KeymapUtil.getShortcutText(it))
+      shortcutLabel.border = BorderFactory.createEmptyBorder(0, 0, 0, 10)
```

Comment:
Pls double check that works correctly for this matrix

| | New UI | Old UI |
| --- | --- | --- |
| Light Theme | 1 | 2 |
| Dark Theme | 3 | 4 |

Reply:
Checked - all works fine now 👍